scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/Job.java

```diff
@@ -460,7 +479,9 @@ public boolean isEqual(Job job) {
                cumulatedCoreTime == job.cumulatedCoreTime && parentId == job.parentId &&
                childrenCount == job.childrenCount && numberOfNodes == job.numberOfNodes &&
                numberOfNodesInParallel == job.numberOfNodesInParallel &&
-               submissionMode != null ? submissionMode.equals(job.submissionMode) : job.submissionMode == null;
+               submissionMode != null ? submissionMode.equals(job.submissionMode)
+                                      : job.submissionMode == null && label != null ? label.equals(job.label)
+                                                                                    : job.label == null;
```

Comment:
Instead of a ternary can't it be  `label == null || label.equals(job.label)` ?
It looks to me that it does the same test but it's a bit more readable

Same for submissionMode

Reply:
it's not enought label == null. If label == null, job.label should be null also. 
So it can be translated (label == null && job.label == null) || label.equals(job.label)